ansible/roles/dev/tasks/main.yml

```diff
@@ -135,8 +135,24 @@
       - app
       - common
       - exceptions
+      - plugin
       - tasking
 
+# We don't currently have a common method for installing plugins for development. In Pulp 2, it was
+# "run pulp-dev.py -I", so maybe that's what we'll do for pulp 3 as well. For now, since only the
+# file plugin works with "python setup.py develop", we'll just try that
+# exists, installing plugins to the platform virtualenv
+- name: Install Plugin packages into Platform Virtualenv
+  command: "{{ pulp_venv_dir }}/pulp/bin/python setup.py develop"
```

Comment:
I'm curious why the other Pulp packages need to be installed system wide, but pulp_file only needs to be in a virtualenv.